needy/needy.py

```diff
@@ -70,6 +73,8 @@ def command(self, cmd, environment_overrides={}):
         environment_overrides['PWD'] = current_directory()
         env = os.environ.copy()
         env.update(environment_overrides)
+        if self.debug():
+            print(Fore.YELLOW + 'command: {}'.format(cmd) + Fore.RESET)
```

Comment:
i think i would prefer to exclude the "command: " prefix since we're highlighting this line

but really, i would have added this a long time ago if it weren't for a few odd cases. what does this do when the command isn't a single string, but a list instead?